### ISSUE TITLE
bat mishighlights Users that start with digits in SSH config 

### ISSUE THREAD

**Issue by sumanthratna** [sharkdp/bat#984]
Related: https://github.com/sharkdp/bat/pull/582

**What version of `bat` are you using?**
bat 0.15.1

**Describe the bug you encountered:**
My `User` for my SSH `Host`s start with a numeric character. `bat` shows the digits in a different color than the alpha characters:
![Screen Shot 2020-05-11 at 22 38 57](https://user-images.githubusercontent.com/31281983/81632474-61b9b180-93d8-11ea-918c-8ec7dff57c99.png)

**Describe what you expected to happen?**
The entire `User` should be colored as tan, not just the `sratna`. Note that GitHub also misrenders the `User`: https://github.com/sumanthratna/dotprophet/blob/50781d97425d37b35a3cfbfb4e72abdb7873325d/sshconfig. 

**How did you install `bat`?**
Homebrew

---

system
------

**$ uname -srm**
Darwin 19.5.0 x86_64  

**$ sw_vers**
ProductName:	Mac OS X  
ProductVersion:	10.15.5  
BuildVersion:	19F72f  

bat
---

**$ bat --version**
bat 0.15.1  

**$ env**

bat_config
----------

bat_wrapper
-----------

No wrapper script for 'bat'.

bat_wrapper_function
--------------------

No wrapper function for 'bat'.

No wrapper function for 'cat'.

tool
----

**$ less --version**
less 487 (POSIX regular expressions)

**Comment by sharkdp** [comment]
Thank you for reporting this.

This needs to be fixed upstream in https://github.com/robballou/sublimetext-sshconfig

**Comment by sumanthratna** [comment]
It looks like this got fixed: https://github.com/robballou/sublimetext-sshconfig/issues/7#event-3329426974. I'll close this issue once the next release of `bat` is published.

**Comment by michaelblyons** [comment]
The sysadmin in me wants to clean up your `ssh_config`.

``` sh
Host tj
    HostName jumpbox.example.com
    User you

Host zoidberg director snowy
    User you
    ProxyJump tj

Host *
    # The things from the top of your file, except
    # most of them are already defaults.
    ServerAliveInterval 120
```

or even

``` sh
Host tj
    HostName jumpbox.example.com

Host zoidberg director snowy
    ProxyJump tj

Host *.example.com
    User you

Host *
    # The things from the top of your file, except
    # most of them are already defaults.
    ServerAliveInterval 120
```

**Comment by sharkdp** [comment]
> It looks like this got fixed: [robballou/sublimetext-sshconfig#7 (comment)](https://github.com/robballou/sublimetext-sshconfig/issues/7#event-3329426974).

:+1: 

I'm closing the ticket now, but feel free to re-open this, if this should still be an issue

**Comment by sumanthratna** [comment]
I just tried `bat ~/.ssh/config` and `bat` isn't showing any syntax highlighting. Should I create a new issue for this or does this sound like an issue with https://github.com/robballou/sublimetext-sshconfig?

**Comment by sharkdp** [comment]
> I just tried `bat ~/.ssh/config` and `bat` isn't showing any syntax highlighting. Should I create a new issue for this or does this sound like an issue with https://github.com/robballou/sublimetext-sshconfig?
> 
> **EDIT**: I installed from `brew`; I'm on `bat 0.15.1`.

That should work and sounds like some kind of bug. It would be great if you could open a new ticket. (I can not reproduce it with 0.15.1)

**Comment by sharkdp** [comment]
This issue should be fixed in [bat v0.15.2](https://github.com/sharkdp/bat/releases/tag/v0.15.2).

**Comment by sumanthratna** [comment]
I don't think this has been fixed for me. The output looks the same as that in the screenshot in the original post.

**Comment by michaelblyons** [comment]
I don't think it's an upstream problem anymore. There's [a unit test](https://github.com/robballou/sublimetext-sshconfig/blob/st3-3.6.0/Tests/syntax_test_client.ssh_config#L36-L38) for your username. It passes. 🤷‍♂️

**Comment by sumanthratna** [comment]
This is strange; the submodule for sublimetext-sshconfig in this repo (sharkdp/bat) has been updated: https://github.com/sharkdp/bat/commit/257f978cebab7942b813cc3bd5ad74a833d35b97.

**Comment by sharkdp** [comment]
Oh, this is my fault. I forgot to rebuild the binary caches. I will put out a new release soon.

**Comment by sharkdp** [comment]
Should be fixed in [v0.15.3](https://github.com/sharkdp/bat/releases/tag/v0.15.3).

**Comment by sumanthratna** [comment]
It works in v0.15.3!
